express/src/lib/publicKeyUtils.ts

```diff
@@ -7,18 +7,16 @@ export default function getAuthApiCompliantPublicKey(publicKey: string): string
     // assume PEM with headers
     try {
         createPublicKey({key: publicKey, format: "pem"});
-        return publicKey.replace(BEGIN, "").replace(END, "").replace("\n", "").replace("\r", "");
-    } catch (err) {
-        console.error(`Failed to convert\n${publicKey}`);
-    }
-
+        return publicKey.replace(BEGIN, "").replace(END, "").replace(/\n/g, "").replace(/\r/g, "");
```

Comment:
`.replace(/\n/g, "").replace(/\r/g, "")` could be `.replace(/[\n\r]/g, "")` AFAIK 🤓

Reply:
I've taken your approval and merged.  I make you right on the regex but future me will take the spoon feeding and be grateful.